src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1756,7 +1756,20 @@ components:
           type: string
         error_message:
           description: |
-            An error message received for a rejection or failure during transaction execution.
+            The initial error message received for a rejection or failure during transaction execution.
+            This will typically be the useful error message, explaining the root cause of the issue.
+            Please note that presence of an error message doesn't imply that this payload
+            will definitely reject or fail. This could represent an error during a temporary
+            rejection (such as out of fees) which then gets resolved (e.g. by depositing money
+            to pay the fee), allowing the transaction to be committed.
+          type: string
+          nullable: true
+        latest_error_message:
```

Comment:
Could you please update CHANGELOG.md file with some brief summarization of pending TX changes?

Reply:
Yes, great shout.